packages/jest-reporters/src/__tests__/GitHubActionsReporter.test.ts

```diff
@@ -5,20 +5,29 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import {afterEach, describe, expect, jest, test} from '@jest/globals';
 import type {
   AggregatedResult,
   AssertionResult,
+  Status,
   Test,
   TestCaseResult,
   TestResult,
 } from '@jest/test-result';
 import type {Config} from '@jest/types';
+import {normalizeIcons} from '../../../../e2e/Utils';
 import GitHubActionsReporter from '../GitHubActionsReporter';
 
 afterEach(() => {
   jest.clearAllMocks();
 });
 
+const oldLog = GitHubActionsReporter.prototype.log;
```

Comment:
Feels like some snapshot serializer could do this job. Can be done in follow up PR. Just saying (;

Reply:
or just

```ts
class TestedGitHubActionsReporter extends GitHubActionsReporter {
  log(message) {
    super.log(normalizeIcons(message))
  }
}
```

or some such